examples/contrib/transformers/dataset.py

```diff
@@ -0,0 +1,35 @@
+import torch
+from torch._C import dtype
```

Comment:
Importing private member is not a good practice, I think you can use `torch.dtype` instead ?